### ISSUE TITLE
Game allows uneven matches in survivor, when one 

### ISSUE THREAD

**Issue by MoonPadUSer** [redeclipse-legacy/base#19]
When playing survivor and one of your team-members leaves during a match, you'll play this round with one person less on the team.

**Comment by TheAssassin** [comment]
That is for the current round only, though, right? The server should rebalance in the next round.

**Comment by MoonPadUSer** [comment]
yes, but at least IMO it should rebalance even during the round in this case
